packages/block-editor/src/components/block-toolbar/index.native.js

```diff
@@ -93,16 +94,17 @@ export default function BlockToolbar( { anchorNodeRef } ) {
 		<>
 			{ isValidAndVisual && (
 				<>
-					<BlockSettingsButton.Slot>
-						{ /* Render only one settings icon even if we have more than one fill - need for hooks with controls. */ }
-						{ ( fills = [ null ] ) => {
-							if ( ! fills?.length > 0 ) {
-								return null;
-							}
-							return fills[ 0 ];
-						} }
-					</BlockSettingsButton.Slot>
-
+					<ToolbarGroup>
```

Comment:
Nice that it was just wrapping the element! I was thinking if we should have it in the [component instead](https://github.com/WordPress/gutenberg/blob/771e5d2ef5132107b400e440c7f54ca05549fb14/packages/block-editor/src/components/block-settings/button.native.js#L11-L18)? To be consistent with the other block controls, what do you think?

Reply:
> I was thinking if we should have it in the [component instead](https://github.com/WordPress/gutenberg/blob/771e5d2ef5132107b400e440c7f54ca05549fb14/packages/block-editor/src/components/block-settings/button.native.js#L11-L18)?

Yep, that's even better. I've updated the code. 👍